src/Components/TaskOne/InputBox.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { default as List } from './List' //why do we need to use default keyword while importing
+class InputBox extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = { value: '' };
+        this.state = { items: [] };
```

Comment:
this state will replace the above state, instead it should be
`this.state = {
   value: '',
   items: []
}`